src/main/resources/static/surveyor.css

```diff
@@ -0,0 +1,8 @@
+#question {
+    display: grid;
+    border-radius: 15px;
+    border: 2px solid #757575;
+    padding: 20px;
```

Comment:
Minor optional suggestion but you can do the padding with percentages instead to make it scalable: https://developer.mozilla.org/en-US/docs/Web/CSS/padding

Reply:
everywhere else we are just using pixels, i wouldnt stress too much about scalability. We arent panning for ios/mobile users to use this. So i think this is fine